src/lib/parameters/parameters.cpp

```diff
@@ -69,6 +69,14 @@ using namespace time_literals;
 #include <uORB/topics/actuator_armed.h>
 #include <uORB/Subscription.hpp>
 
+/* Include functions common to user and kernel sides */
+#include "parameters_common.cpp"
```

Comment:
I prefer having this as a header, you can make most of the methods `static inline`.

Reply:
making these static inline is actually not that simple. The generated px4_parameters.hpp can be included just once, otherwise the tables will get multiplied increasing the code size. And just re-naming the file to .h ends up style checks failing in CI. Need to think about this, what might be the best alternative way to avoid having duplicate code.

Reply:
Ok not a big deal, it's just a bit of an anti-pattern.